README.md

```diff
@@ -1,70 +1,50 @@
-# Getting Started with Create React App
+![](https://img.shields.io/badge/Microverse-blueviolet)
 
-This project was bootstrapped with [Create React App](https://github.com/facebook/create-react-app).
+# Vehicles App
```

Comment:
## Optional
- Please add lines describing what is the project about. Here will be the first page "a recruiter " will read to understand your project and it can help you to land a first Job contract